### PR TITLE
DM-36204: Add alt to metadata table

### PR DESCRIPTION
**Pull request by mfisherlevine**

### CODE REVIEW

File: python/lsst/rubintv/production/rubinTv.py
Comment (timj) [inline python/lsst/rubintv/production/rubinTv.py:535]: Assigning a string to something that should be a float does make me wince a bit when thinking about a `mypy` future. Is there a reason this is not `None` or `NaN`?

**Review by timj — APPROVED** [review]
Looks okay to me. I don't understand the `"null"` usage though.

**Comment by timj** [comment]
You really need to add some branch protections and GitHub actions to this repo, even if it's just copying over the lint.yaml from science pipelines.

File: python/lsst/rubintv/production/rubinTv.py
Comment (mfisherlevine) [reply python/lsst/rubintv/production/rubinTv.py:535]: Yeah, fair enough. I think `None` is the right answer here, I just need to check that that does indeed get rendered as `null` like everything else on the front end. I'll do that tomorrow morning once we're not on sky, and will merge with that change assuming it works. I don't think any of this will ever be mypyed though.

**Comment by mfisherlevine** [comment]
> You really need to add some branch protections and GitHub actions to this repo, even if it's just copying over the lint.yaml from science pipelines.

Yeah, this is a very good shout. I'll take a look at that in the morning as well.

File: python/lsst/rubintv/production/rubinTv.py
Comment (mfisherlevine) [reply python/lsst/rubintv/production/rubinTv.py]: `None` works perfectly. Merging now.

**Pull request merged**
